### PR TITLE
Release 0.5.0

### PR DESCRIPTION
**Pull request by nforro**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# 0.5.0
+
+- Strict optional typing is now enforced (#68)
+- Fixed deduplication of tag names (#69)
+- Sources and patches can now be removed by number (#69)
+- Number of digits in a source number is now expressed the same way as packit does it (#69)
+- Empty lines are now compressed when deleting tags (#69)
+- Added convenience property for getting texts of tag comments (#69)
+- Added convenience method for adding a patch (#69)
```
Comment (FrNecas) [inline CHANGELOG.md:9]: You are missing one of the changes from #69 , I think. This commit, to be precise: https://github.com/packit/specfile/pull/69/commits/f040a3f5aaab772b36346858b889ae93330defcb

**Review by FrNecas — APPROVED** [review]
One missing change (which was added later than this PR was created), otherwise looks good

File: CHANGELOG.md (hunk 1 above)
Comment (nforro) [reply CHANGELOG.md:9]: That's on purpose, I didn't think it was worth mentioning. Do you think I should mention it?

Comment (FrNecas) [reply CHANGELOG.md:9]: Ah, I see. It's not necessary then :)

**Pull request merged**